### ISSUE TITLE
Switch CI/CD build to use more deterministic dependency install

### ISSUE THREAD

**Issue by mgwalker** [18F/cms-hitech-apd#454]
Currently we use `npm install` to install dependencies in our CI/CD builds (and after #452, our deployed libraries as well).  This does some fancy dance moves to determine which versions to install, and we can end up with different versions every time we CI runs.

We can switch to `npm ci` (after #453), which installs exclusively from the package lockfile.  It's a faster install because there's less version searching and matching to do, but also we can be sure that what's used in CI matches whatever we last installed locally.

As a nice bonus, `npm ci` never modifies the package files where as `npm install` can modify the lockfile even if nothing actually changed.  The result is that `npm install` can mess up the CircleCI cache by changing the hash of the lockfile.  `npm ci` won't do that.  :tada:

**Comment by NAretakis** [comment]
@mgwalker is this a thing we should still do? Maybe part of the refinement milestone? Or is this dependent on infrastructure?

**Comment by mgwalker** [comment]
Oh!  This is done.
